backends/rapidpro/msg.go

```diff
@@ -518,6 +518,8 @@ type DBMsg struct {
 	SessionWaitStartedOn_ *time.Time `json:"session_wait_started_on,omitempty"`
 	SessionStatus_        string     `json:"session_status,omitempty"`
 
+	Flow_ json.RawMessage `json:"flow,omitempty"`
```

Comment:
no reason to not parse it all in one go.. you can even use a anon struct like 

```
Flow_ *struct {
    UUID string `json:"uuid"`
    Name string `json:"name"`
} `json:"flow"`
``

Reply:
What type will `Flow()` will return there?

Reply:
never mind no need for anymore

Reply:
if you need a type just add something like `MsgFlowRef`

Reply:
I am confused on how to use that I will try that again tomorrow